configure.in

```diff
@@ -749,9 +749,10 @@ WX_ARG_FEATURE(threads,     [  --enable-threads        use threads], wxUSE_THREA
 if test "$wxUSE_MSW" = 1 ; then
 WX_ARG_DISABLE(dbghelp,     [  --enable-dbghelp        use dbghelp.dll API (Win32 only)], wxUSE_DBGHELP)
 WX_ARG_ENABLE(iniconf,      [  --enable-iniconf        use wxIniConfig (Win32 only)], wxUSE_INICONF)
-WX_ARG_FEATURE(regkey,      [  --enable-regkey         use wxRegKey class (Win32 only)], wxUSE_REGKEY)
 fi
 
+WX_ARG_FEATURE(regkey,      [  --enable-regkey         use wxRegKey class (Win32 only)], wxUSE_REGKEY)
```

Comment:
wxRegKey isn't available on all platforms. It (and probably wxIniConfig, and dbghelp) was wxMSW-port dependent and probably should become Windows-platform dependent.